{"name": "test_logging_in_with_Invalid_credentials", "status": "passed", "attachments": [{"name": "stdout", "source": "d503de6f-5dc4-4b6f-944d-e39a144a116b-attachment.txt", "type": "text/plain"}], "start": 1720110563105, "stop": 1720110563105, "uuid": "f61203c2-df8d-4887-a4f2-d86ff39d9542", "historyId": "40fc0dfae200b9231ec55ef92e0c5e51", "testCaseId": "40fc0dfae200b9231ec55ef92e0c5e51", "fullName": "python.selenium.dr.pytest_test_programming.test_pytestfixtures#test_logging_in_with_Invalid_credentials", "labels": [{"name": "parentSuite", "value": "python.selenium.dr.pytest_test_programming"}, {"name": "suite", "value": "test_pytestfixtures"}, {"name": "host", "value": "DESKTOP-UI2KGFP"}, {"name": "thread", "value": "13356-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "python.selenium.dr.pytest_test_programming.test_pytestfixtures"}]}